### PR TITLE
ISSUE-856 Event logging for test runner should log output events

### PR DESCRIPTION
**Pull request by sameer79**

### CODE REVIEW

File: webservice/src/main/resources/app/scripts/containers/Streams/TestRunComponents/TestSourceNodeModal.jsx [hunk 1]
```diff
@@ -179,7 +179,7 @@ class TestSourceNodeModal extends Component{
     const {showInputError,sourceNodeArr, testName} = this.state;
     let validate = false,validationArr = [];
     _.map(sourceNodeArr, (source,i) => {
-      if(source.records === '' || source.repeatTime === '' || source.repeatTime === undefined || source.records === undefined || testName === ''){
+      if(source.records === '' || source.repeatTime === '' || (source.repeatTime === undefined || source.repeatTime === 0 )|| source.records === undefined || testName === ''){
```
Comment (shahsank3t) [inline webservice/src/main/resources/app/scripts/containers/Streams/TestRunComponents/TestSourceNodeModal.jsx]: It's ok to have 0 as Repeat when the user just wants to run the test on the provided record.
Let's not restrict the user to enter more than 0 as value.

Comment (HeartSaVioR) [inline webservice/src/main/resources/app/scripts/containers/Streams/TestRunComponents/TestSourceNodeModal.jsx]: Maybe restricting it and set default value to 1 would be clearer.

**Comment by shahsank3t** [comment]
+1 after that nit suggestion is addressed.

**Comment by sameer79** [comment]
Address the suggestion

**Pull request merged**